valhalla/mjolnir/ferry_connections.h

```diff
@@ -32,14 +32,18 @@ constexpr uint32_t kPermanent = 1;
 // be "adjacent" to an node that is permanently labeled.
 constexpr uint32_t kTemporary = 2;
 
+constexpr uint32_t kFerryUpClass = static_cast<uint32_t>(baldr::RoadClass::kPrimary);
+
```

Comment:
the road class was already a constant, but was passed through the whole function chain only to end up where it's needed. rather make it a real constanct